javascripts/govuk_toolkit.js

```diff
@@ -1 +1,13 @@
 // = require_tree ./govuk
+
+(function () {
+  if (window.GOVUK.analytics && window.GOVUK.analytics.trackEvent) {
+    if (window.location.origin === 'https://gov.uk') {
+      GOVUK.analytics.trackEvent('Toolkit', 'stop-scrolling-at-footer.js', {
```

Comment:
I think this is a copy paste error from the previous file? `stop-scrolling-at-footer.js`?

Reply:
Fixed.